di_website/publications/blocks.py

```diff
@@ -49,3 +48,16 @@ class AudioMediaStreamBlock(StreamBlock):
     media = AudioMediaBlock()
 
     required = False
+
+class PublicationBlockQuote(StructBlock):
+    """
+    Custom `StructBlock` that allows render text in a blockquote element
+    """
+    text = TextBlock()
+    source = TextBlock(required=False, help_text='Who is this quote acredited to?')
+    center = BooleanBlock(default=False, required=False)
+
+    class Meta:
+        icon = 'fa-quote-right'
+        template = 'blocks/publication_blockquote.html'
+
```

Comment:
This is an entirely new block, which is redundant ... u only need to specify a different template when using the already existing block i.e `BlockQuote(template='path/to/template')`